life_expectancy/tests/test_loading_saving.py

```diff
@@ -1,18 +1,19 @@
 """Tests for the loading_saving module"""
 from unittest.mock import patch
 import pandas as pd
-
-from life_expectancy.loading_saving import load_data, save_data
-from . import FIXTURES_DIR
+from life_expectancy.data_cleaning import CSVDataCleaner
+from life_expectancy.loading_saving import save_data
+from life_expectancy.env_variables import  CSV_TEST_FILE_PATH
 
 
 def test_load_data(eu_life_expectancy_loaded_dataframe_test):
     """Test the load_data method that loads the raw test dataframe"""
 
-    loaded_test_data = load_data(file_path=FIXTURES_DIR / 'eu_life_expectancy_raw_test.tsv')
+    data_loader = CSVDataCleaner(file_path=CSV_TEST_FILE_PATH)
```

Comment:
Might be interesting to have a test for the json load data.

Reply:
Forgot abou that one. Will add it